Regression/WarpX-tests.ini

```diff
@@ -209,6 +209,22 @@ compileTest = 0
 doVis = 0
 analysisRoutine = Examples/Tests/btd_rz/analysis_BTD_laser_antenna.py
 
+[collider_diagnostics]
+buildDir = .
+inputFile = Examples/Tests/collider_relevant_diags/inputs_3d_multiple_particles
+runtime_params = warpx.abort_on_warning_threshold=high
```

Comment:
Why do we set this at `high` ? Does the simulation generate some `low` or `medium` priority warnings?

Reply:
Yes, I have added two low priority warnings for the `ColliderRelevant` diagnostic to let the user know that the colliding species should propagate along `z` and that the various `chi` values are computed on the coarsest mesh refinement level.